test/playwright/e2e/search-navigation-old.spec.ts

```diff
@@ -24,6 +24,9 @@ test.describe("search history navigation", () => {
 
     // Apply a filter
     await page.click("#modification")
+    // There is a debounce of 500 ms when choosing a filter.
+    //we need to wait for the page to get reloaded before procceding the test
```

Comment:
```suggestion
    // we need to wait for the page to reload before running the test
```

here is a small grammatical suggestion